compiler/luci/service/src/CircleShapeSignatureInferenceHelper.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Service/CircleShapeSignatureInferenceHelper.h"
+
+namespace ShapeSignatureInferenceHelper
+{
+
+luci::ShapeSignature signature_of_Nth_input(const luci::CircleNode *node, uint32_t index)
```

Comment:
do we need `_Nth_` in name? why not just `input_signature`  ? or `signature_of_input` ?

Reply:
I will change to `signature_of_input` :)